bpf/cpu/cpu.bpf.c

```diff
@@ -1097,6 +1104,14 @@ int profile_cpu(struct bpf_perf_event_data *ctx) {
     return 0;
   }
 
+  // 2. We did not have unwind information, let's see if we can unwind with frame
+  // pointers.
+  if (has_fp(unwind_state->bp)) {
+    add_stack(ctx, pid_tgid, STACK_WALKING_METHOD_FP, NULL);
+    return 0;
+  }
+
+  // 3. Request unwind information.
   request_unwind_information(ctx, user_pid);
```

Comment:
Thanks for all the numbered comments here!

Reply:
:D hopefully things will be clearer and this will also prevent accidental reorders!